manager/controllers/mockup/policymanager.go

```diff
@@ -94,6 +98,15 @@ func (s *MockPolicyManager) GetPoliciesDecisions(ctx context.Context, in *pb.App
 			Decisions: operationDecisions})
 	}
 
-	return &pb.PoliciesDecisions{ComponentVersions: externalComponents,
-		DatasetDecisions: dataSetWithActions}, nil
+	result := &pb.PoliciesDecisions{ComponentVersions: externalComponents,
+		DatasetDecisions: dataSetWithActions}
+
+	policyManagerResp, _ := connectors.ConvertGrpcRespToOpenAPIResp(result)
+
+	res, err := json.MarshalIndent(policyManagerResp, "", "\t")
+	log.Println("err :", err)
+	log.Println("policyManagerResp: created from convGrpcRespToOpenApiResp")
+	log.Println("marshalled response:", string(res))
```

Comment:
Why are you printing the error and then ignoring it? shouldn't it just be returned?

Reply:
Fixed. Please review. Thanks.